modules/analysis/rosbag-image-pipeline/modulestack.template

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
It seems like you dont use this boilerplate CFN template. If yes, can you remove it?

Reply:
I removed the file completely